addon/instance-initializers/flexberry-logging.js

```diff
@@ -10,6 +10,16 @@ import Ember from 'ember';
   @for FlexberryLoggingService
  */
 export function initialize(applicationInstance) {
+
+  /**
+   *  flexberry-logging service for transmit error/warning/log/info/debug/deprecation messages to  store and save its on server as i-i-s-caseberry-logging-objects-application-log object
+   *
+   * @property flexberryLogging
+   * @type Object Ember.Service
+   * @default service:flexberry-logging
+   */
+  var flexberryLogging = applicationInstance.lookup('service:flexberry-logging');
```

Comment:
As far as I know we can inject services like this:

Ember.inject.service('objectlistview-events'). Can't we use this variant?